ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -151,28 +151,34 @@ if [[ "${CLUSTER_TYPE}" == gcp ]]; then
 fi
 
 function upgrade() {
-    set -x
-    TARGET_RELEASES="${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE:-}"
+    set -x &&
```

Comment:
Could we leave a comment above each of these functions with a reminder to keep the chained &&'s? For whatever unfortunate person that isn't us touches it next.

Reply:
I've added comments to each of the two functions that are currently called from an AND-OR list in ad4354a4de -> 904bc3a557.